OWNERS

```diff
@@ -17,5 +17,4 @@ emeritus_maintainers:
   - ncdc
 
 reviewers:
-  - cluster-api-maintainers
-  - cluster-api-reviewers
+  - cluster-api-operator-reviewers
```

Comment:
what do you think about creating a separate OWNERS file in exp/operator so it's scoped to only the operator files?

Reply:
ah I see Fabrizio already the same thing commented above https://github.com/kubernetes-sigs/cluster-api/pull/4260#pullrequestreview-605599683

Reply:
PTAL. Does this make more sense? I tried to read the docs beforehand but am not too familiar with the OWNER structure.